ironfish/src/webApi.ts

```diff
@@ -158,7 +158,18 @@ export class WebApi {
     return await axios
       .get<ApiUser>(`${this.host}/users/find`, options)
       .then((r) => r.data)
-      .catch(() => null)
+      .catch((error: AxiosError<{ code: string; message?: string }>) => {
+        if (error.response) {
+          const { status } = error.response
+          if (status >= 500) {
```

Comment:
We can use [`isResponseServerError(error.response)`](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/response.ts#L13-L15)